production/loki-mixin/config.libsonnet

```diff
@@ -11,6 +14,20 @@
 
     // The label used to differentiate between different clusters.
     per_cluster_label: 'cluster',
+    per_namespace_label: 'namespace',
+    per_job_label: 'job',
+
+    // Grouping labels, to uniquely identify and group by {jobs, clusters}
+    job_labels: [$._config.per_cluster_label, $._config.per_namespace_label, $._config.per_job_label],
+    cluster_labels: [$._config.per_cluster_label, $._config.per_namespace_label],
+
+    // Each group prefix is composed of `_`-separated labels
+    group_prefix_jobs: makePrefix($._config.job_labels),
+    group_prefix_clusters: makePrefix($._config.cluster_labels),
+
+    // Each group-by label list is `, `-separated and unique identifies
+    group_by_job: makeGroupBy($._config.job_labels),
+    group_by_cluster: makeGroupBy($._config.cluster_labels),
```

Comment:
IIUC this allows for any of the labels that are defaulted to be in `x_labels`, for example `cluster, namespace, job`, to actually be removed, in case someone only wants to select on `namespace, job` for example?

Reply:
That is correct.